CONTRIBUTING.rst

```diff
@@ -0,0 +1,153 @@
+How to contribute to Flask-SQLAlchemy
+=====================================
+
+Thank you for considering contributing to Flask-SQLAlchemy!
+
+Support questions
+-----------------
+
+Please, don't use the issue tracker for this. Use one of the following
+resources for questions about your own code:
+
+* The IRC channel ``#pocoo`` on FreeNode.
+* The IRC channel ``#python`` on FreeNode for more general questions.
+* The mailing list flask@python.org for long term discussion or larger issues.
```

Comment:
Do we want FSA mail to go to flask@python.org?

Reply:
Do we want/need a mailing list at all? I just looked at the archive (I'm not even subscribed myself..) and the Flask list is incredibly low-volume.

Reply:
I confirmed this with @davidism but I'm happy to change it if you all decide to go in a different direction.